samples/core/loop_parameter/loop_parameter_test.py

```diff
@@ -17,29 +17,8 @@
 from ...test.util import run_pipeline_func, TestCase, NEEDS_A_FIX
 
 run_pipeline_func([
-    # TODO(v2-compatible): fix this sample for v2 mode.
-    # V2_COMPATIBLE mode fails with:
-    # File "/Users/gongyuan/kfp/pipelines/sdk/python/kfp/compiler/v2_compat.py", line 108, in update_op
-    # artifact_info = {"fileInputPath": op.input_artifact_paths[artifact_name]}
-    # KeyError: 'text'
-    #
-    # And another error:
-    # This step is in Error state with this message: withParam value could not
-    # be parsed as a JSON list:
-    # {
-    #   "id":"733",
-    #   "typeId":"114",
-    #   "uri":"gs://gongyuan-test/kfp/output/pipeline-with-loop-parameter/pipeline-with-loop-parameter-pn8nj/pipeline-with-loop-parameter-pn8nj-1197421549/data",
-    #   "createTimeSinceEpoch":"1617689193656",
-    #   "lastUpdateTimeSinceEpoch":"1617689193656"
-    # }
-    # TestCase(
-    #     pipeline_func=my_pipeline,
-    #     mode=kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE,
-    #     verify_func=NEEDS_A_FIX,
-    # ),
     TestCase(
         pipeline_func=my_pipeline,
-        mode=kfp.dsl.PipelineExecutionMode.V1_LEGACY,
```

Comment:
@chensun I think we want to keep testing the sample works for both V1 and V2, so there should be two test cases here.

Reply:
I see. I misunderstood. Fixed.